server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -392,6 +392,7 @@ public void test_write_to_subscribed_table_is_allowed_after_dropping_subscriptio
             "The relation \"doc.t1\" doesn't support or allow INSERT operations."
         );
 
+        executeOnSubscriber("ALTER TABLE doc.t1 CLOSE"); // TODO - make it part of DROP SUB
```

Comment:
Will remove it once I do another iteration with adapting TransportCloseTable (extract main logic + make it accept list)